### PR TITLE
Adds Unathi

### PR DESCRIPTION
**Pull request by Takykardi**
Well, hopefully.

As it says on the tin, adds Unathi. Radioactive desert lizards, copy-pasted straight from Bay12 back to this Bay downstream.

### CODE REVIEW

**Comment by miniusAreas** [comment]
haha lizard go hiss

god bless you torque

**Comment by MistakeNot4892** [comment]
Indentation is borked - you have a line somewhere using spaces instead of tabs.
Other fails seem to be map related rather than related to this PR.

File: mods/general_content/GeneralContent.dme [hunk 1]
```diff
@@ -17,3 +18,12 @@
 #include "sprite_accessory/overwrites_head.dm"
 
 #endif
+// BEGIN_INTERNALS
```
Comment (MistakeNot4892) [inline mods/general_content/GeneralContent.dme]: This chunk needs to be removed, it is generated by DM and messes with things.

File: mods/general_content/species/lizard.dm [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+	husk_icon = 'icons/unathi/husk.dmi'
+	preview_icon = 'icons/unathi/preview.dmi'
+	bandages_icon = 'icons/mob/bandage.dmi'
+	tail = "sogtail"
+	tail_animation = 'icons/unathi/unathi_tail.dmi'
+	limb_blend = ICON_MULTIPLY
+	tail_blend = ICON_MULTIPLY
+	hidden_from_codex = FALSE
+
+	//unarmed_types = list(/datum/unarmed_attack/stomp, /datum/unarmed_attack/tail, /datum/unarmed_attack/claws, /datum/unarmed_attack/punch, /datum/unarmed_attack/bite/sharp)
```
Comment (MistakeNot4892) [inline mods/general_content/species/lizard.dm]: `unarmed_attacks = list(/decl/natural_attack/honk)`

File: mods/general_content/species/lizard.dm [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+	cold_discomfort_level = 290
+	cold_discomfort_strings = list(
+		"You feel chilly.",
+		"You feel sluggish and cold.",
+		"Your scales bristle against the cold."
+		)
+	breathing_sound = 'sound/lizard.ogg'
+
+	/*base_auras = list(
```
Comment (MistakeNot4892) [inline mods/general_content/species/lizard.dm:76]: You should be able to straight up copypaste the aura definition from Bay to Neb and have this work.

File: mods/general_content/species/lizard.dm [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+			list(/decl/emote/audible/grunt, /decl/emote/audible/groan, /decl/emote/audible/wheeze, /decl/emote/audible/hiss) = 50,
+			list(/decl/emote/audible/grunt, /decl/emote/audible/groan, /decl/emote/audible/hiss) = 20,
+		)
+
+/datum/species/unathi/equip_survival_gear(var/mob/living/carbon/human/H)
+	..()
+	H.equip_to_slot_or_del(new /obj/item/clothing/shoes/sandal(H),slot_shoes)
+
+/datum/species/unathi/get_bodytype(var/mob/living/carbon/human/H)
+	return SPECIES_UNATHI
```
Comment (MistakeNot4892) [inline mods/general_content/species/lizard.dm]: This needs to return a bodytype, not a species name. Refer to https://github.com/EscapePodNo5/EPN5-Neb/blob/dev/code/modules/species/species.dm#L24

File: mods/general_content/species/lizard.dm (hunk 3 above)
Comment (Takykardi) [reply mods/general_content/species/lizard.dm:76]: The Unathi loredev requested I have this disabled for the meantime pending lore changes.

**Pull request merged**